### PR TITLE
cores updates

### PR DESCRIPTION
**Pull request by schmurtzm**
**Update .atari800.cfg**

typo fix : ATARIOSA.ROM in upcase , atari800 core is not key sensitive with bios.

 **Update OpenBOR with Steward-Fu version**

Shrinked version of  OpenBor from Steward-Fu (openbor_20220422) :
Doesn't requires Parasyte framework, only essential files have been kept. Replace the existing OpenBor integrated in Onion which was broken.

 **Amiga replace puae core with puae 2021**

Recommended for low hardware, should fix slowdowns with parallax scrolling effect games and fixes a lot of things like compatibility and auto-zoom mode.
Chd files now works perfectly.

Builded from https://github.com/libretro/libretro-uae/tree/2.6.1
Commit c68404cf2e83f1136a45b7b3d3f775c619d9d35f   (Sep 12, 2022)

 **add amiga capsimg.so (ipf library)**

capsimg.so is required to load .ipf roms

This one is compatible and comes from :
https://github.com/midwan/amiberry/releases/download/v2.25/amiberry-rpi-v2.25.zip

### CODE REVIEW

File: static/packages/RApp/Game engine - Open Beats of Rage/RApp/OpenBOR/launch.sh [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
-progdir=`dirname "$0"`
+mydir=`dirname "$0"`
 
-cd $progdir
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$progdir
-HOME=/mnt/SDCARD $progdir/OpenBOR.ss
\ No newline at end of file
+
+export HOME=$mydir
+export PATH=$mydir:$mydir/bin:$PATH
```
Comment (Aemiii91) [inline static/packages/RApp/Game engine - Open Beats of Rage/RApp/OpenBOR/launch.sh:6]: This is not needed. The directory doesn't have a `bin` folder.

**Comment by Aemiii91** [comment]
I think you should move all the libs into `.tmp_update/lib`, so it can be shared across the system, and we avoid having duplicates.

**Comment by schmurtzm** [comment]
> I think you should move all the libs into `.tmp_update/lib`, so it can be shared across the system, and we avoid having duplicates.

Yes 👍, I'll compare it with the one for FFplay and make a new commit.

**Comment by Aemiii91** [comment]
It would be nice if we could just take all libs we need from parasyte and add them to `.tmp_update/libs` .. I'm currently working on the new ports collection, and I see some of the ports also need SDL2 for example (e.g. C-Dogs)

**Comment by schmurtzm** [comment]
About PUAE2021 I have different feedbacks, on discord gklka said : 

> Setting the correct system to emulate helps a lot, and A500 games are almost 100% playable. I was able to enjoy many of them which I reported slow previously.

What are the games you test and what is your testing procedure ?

I've compiled some KMFD xtreme core for multiple platforms ( LudicrousN64-Xtreme on orange pi, mame 2003 xtreme for Orange pi and for the Miyoo Mini , vice xtreme for the Miyoo Mini, pcsx xtreme for Miyoo Mini) and I'm now very skeptical on the improvements of these cores... I also watch kmfd video with interest but when you look to his different forks there not so much interesting commits...

**Pull request closed without merging**